Content lifecycle management: patch sharing in Uyuni (#899)
uyuni-project/uyuni-docs: modules/administration/pages/content-lifecycle.adoc

Building applies filters to the attached sources and clones them to the first environment in the project.

> You can clone vendor channels multiple times within your organization, which results in having two or more content projects with the same source vendor channels.

Reviewer: ```suggestion
You can clone vendor channels multiple times within your organization, which results in two or more content projects having the same vendor channels.
```

Author: In the end I've rewritten this sentence. Its current form does not make sense (it says cloning multiple channels results in multiple projects having the same vendor channel, but it's actually the other way around: 2 projects having same vendor channels results in multiple clones). I've tried to reduce this to bare minimum in order to prevent any unclarities.

In this case, {productname} does not create new patch clones for each cloned channel. 
Instead, a single patch clone is shared between all of your cloned channels.

> This can cause problems if a vendor modifies a patch; for example if the patch is retracted, or the packages within the patch are changed.

Reviewer: ```suggestion
This can cause problems if a vendor modifies a patch; for example, if the patch is retracted, or the packages within the patch are changed.
```

When you build one of the content projects, all the channels that share the cloned patch are synchronized with the original by default, even if the channels are in other environments of your content project, or other content project channels in your organization.
You can change this behavior by turning off automatic patch synchronization in your organization settings. 

> To manually synchronize the patch later for all channels sharing the patch, navigate to menu:Software[Manage > Channels], click the channel you want to synchronize and navigate to the [guimenu]Sync subtab.

Reviewer: ```suggestion
To manually synchronize the patch later for all channels sharing the patch, navigate to menu:Software[Manage > Channels], click the channel you want to synchronize and navigate to the [guimenu]``Sync`` subtab.
```

> Note that even the manual patch synchronization affects all organization channels sharing the patch.

Reviewer: ```suggestion
Even manual patch synchronization affects all organization channels sharing the patch.
```

Building applies filters to the attached sources and clones them to the first environment in the project.

> You can use same vendor channels as sources for multiple content projects.

Reviewer 2: ```suggestion
You can use the same vendor channels as sources for multiple content projects.
```

Building applies filters to the attached sources and clones them to the first environment in the project.

[NOTE]
====
About patch sharing

When you clone a vendor channel multiple times in your organization (e.g. you have 2 content projects with the same source vendor channels), {productname} does not create new patch clones for each cloned channel. Instead of that, a single patch clone is shared between your cloned channels.

One important consequence is, that if the vendor modifies a patch (e.g. the patch gets retracted, or the patch packages change) and you build one content project, all channels sharing the cloned patch be synchronized with the original (even the channels in other environments of your content project, or other content project channels in your organization).

> You can turn the automatic patch synchronization feature off in your organization settings. You can synchronize the patch later manually using the Patch Sync function (menu:Software[Manage > Channels], click the channel you want to synchronize, and follow the menu:Repositories[Sync]).

Reviewer 3: But synchronize a patch will still affect all channels where this patch exists in. Only the point in time when this happens is different.

Author: Ok, maybe it deserves a more detailed explanation. Thanks for the comment!

> [NOTE]
> ====
> About patch sharing
> 
> When you clone a vendor channel multiple times in your organization (e.g. you have 2 content projects with the same source vendor channels), {productname} does not create new patch clones for each cloned channel. Instead of that, a single patch clone is shared between your cloned channels.
> 
> One important consequence is, that if the vendor modifies a patch (e.g. the patch gets retracted, or the patch packages change) and you build one content project, all channels sharing the cloned patch be synchronized with the original (even the channels in other environments of your content project, or other content project channels in your organization).
> 
> You can turn the automatic patch synchronization feature off in your organization settings. You can synchronize the patch later manually using the Patch Sync function (menu:Software[Manage > Channels], click the channel you want to synchronize, and follow the menu:Repositories[Sync]). Note that even manual patch synchronization affects all channels sharing the patch.
> ====

Reviewer 2: ```suggestion
You can clone vendor channels multiple times within your organization, which results in having two or more content projects with the same source vendor channels.
In this case, {productname} does not create new patch clones for each cloned channel. 
Instead, a single patch clone is shared between all of your cloned channels.
This can cause problems if a vendor modifies a patch; for example if the patch is retracted, or the packages within the patch are changed.
When you build one of the content projects, all the channels that share the cloned patch are synchronized with the original by default, even if the channels are in other environments of your content project, or other content project channels in your organization.
You can change this behavior by turning off automatic patch synchronization in your organization settings. 
To manually synchronize the patch later for all channels sharing the patch, navigate to menu:Software[Manage > Channels], and click the channel you want to synchronize.
You can monitor progress by navigating to menu:Repositories[Sync]. 
```

Author: I like the suggested change, but the link to `Sync` is wrong. Sorry, I've messed it up. Also, the `Sync` screen does not allow you to monitor the progress. It only allows you to perform the action. Instead of the last 2 lines,  I'd suggest something like that:

"To manually synchronize the patch later for all channels sharing the patch, navigate to menu:Software[Manage > Channels], click the channel you want to synchronize and navigate to the [guimenu]``Sync`` subtab."

Moreover, I'd like to explicitly mention the effect on all channels, as it was mentioned in my changes: "Note that even the manual patch synchronization affects all channels sharing the patch.". We can even reword it somehow, but I'd  like this information not to be omitted.

Author: (Forgot to add your handle, doing that now @Loquacity  :) )

Author: I've just accepted your suggestion and pushed the changes I mentioned in my latest comment.
